README.md

```diff
@@ -40,14 +42,20 @@ npx @readme/micro@v2.4.0 './*{yaml,yml,json}' --key=$README_MICRO_SECRET
 ### As a Bitbucket Pipeline
 
 ```yaml
-image: node:18
-
+definitions:
+  steps:
+    - step: &readme-micro
+        name: ReadMe Micro
+        image: node:18
+        script:
+          - npx @readme/micro@v2.4.0 '**/*.{yaml,yml,json}' --key=$README_MICRO_SECRET
```

Comment:
could use conditions to run Micro only for this glob, but that'll add complexity when folks are integrating Micro into their `bitbucket-pipelines.yml`.

in an ideal world, this repo would have a Dockerfile, a `pipe.yml`, etc. so Bitbucket could use that Pipe, instead this getting converted to its own docker image before the Pipeline is run.

[the number of lines needed](https://support.atlassian.com/bitbucket-cloud/docs/write-a-pipe-for-bitbucket-pipelines/#Step-6---Write-README.md---your-readme-file:~:text=to%20Slack.-,YAML%20Definition,slack%2Dnotify%3A0.2.1%0A%20%20variables%3A%0A%20%20%20%20WEBHOOK_URL%3A%20%27%3Cstring%3E%27%0A%20%20%20%20MESSAGE%3A%20%27%3Cstring%3E%27%0A%20%20%20%20%23%20DEBUG%3A%20%27%3Cboolean%3E%27%20%23%20Optional.,-Variables) for folks to plug in the Micro pipe in to their `bitbucket-pipelines.yml` file, but that can probably wait until we see Bitbucket customers requesting it.